scripts/README.md

```diff
@@ -1,15 +1,84 @@
-# Using scripts
+## Steps to use scripts
+### Deploy Single dataset with github actions
+The github actions below will automatically build and deploy a single dataset to gh-pages branch. Folow the steps below to achieve this:
 
-## Deploy single dataset with no portal commit
+**Step 1:** In the dataset repository you want to deploy, create a github secret with the name `PORTAL_REPO_NAME` and the value should be the name of the repository. 
```

Comment:
I'd make "Step 1" a heading e.g. h3 - that way i can link it ...

Reply:
Okay